docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/views-view-unformatted--student-profiles.html.twig

```diff
@@ -143,10 +143,11 @@
 
     {% block card_tag %}
         {% if person_blurb|striptags|trim is not empty %}
-            <div class="card__date"><span class="fas fa-quote-left"></span> {{ person_blurb }}</div>
+            <div class="card__date"><span class="fas fa-quote-left"></span> <div class="col-md-4">{{ person_blurb }}</div></div>
```

Comment:
While this is a creative solution, but we probably don't want to use the Bootstrap classes for spacing.   What style treatment are you looking for here?